views/routes/routes.js

```diff
@@ -43,7 +43,9 @@ export const apiRoutes = {
   },
   users: {
     user: (uid) => `${apiPath}/users/${uid}`,
-    preferences: (uid) => `${apiRoutes.users.user(uid)}/preferences`,
+  },
+  users: {
+    user: (uid) => `${apiPath}/users/${uid}`,
```

Comment:
This looks to be a dup of the line above.

Reply:
it has been removed